src/modules/App/components/CustomToaster/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { ClassNames, css, Theme, useTheme } from '@emotion/react'
+
+import { ToastBar, Toaster } from 'react-hot-toast'
+
+import { ToastContextProvider } from '@/common/components/Toast'
+
+const toastStyles = (theme: Theme) => css`
+  top: 20px;
+  font-size: 1rem;
+  font-family: Prompt;
+  border-radius: 4px;
+  padding: 0;
+  padding-left: 10px;
+  border: 1px solid ${theme.palette.divider};
+
+  div[role='status'] {
+    margin: 0;
+  }
+`
+
+export function CustomToaster() {
+  const theme = useTheme()
+
+  return (
+    <ClassNames>
+      {({ css }) => (
+        <Toaster
+          position="top-center"
+          toastOptions={{
+            duration: 99999,
```

Comment:
Why does default duration need to be Infinity?

Reply:
Oops that was for debugging

Reply:
```suggestion
```